vendor/google.golang.org/api/internal/BUILD.bazel

```diff
@@ -13,7 +13,7 @@ go_library(
     deps = [
         "//vendor/golang.org/x/oauth2:go_default_library",
         "//vendor/golang.org/x/oauth2/google:go_default_library",
-        "//vendor/google.golang.org/grpc/naming:go_default_library",
+        "@org_golang_google_grpc//naming:go_default_library",
```

Comment:
tbh, I don't think editing files in the vendor dir is a good idea. I expect that it will be overwritten sooner or later, e.g. when running `go mod vendor`?

Reply:
I thought so too but that was the only thing that helped. 

IIUC the correct solution to this would be to make use of the `go_proto_library` rules? Do you have an idea on how to do this? I couldn't figure this out myself...

Reply:
You are right. Actually the combination of 

```
go mod vendor
bazel run //:gazelle
```

overrides the changes in `vendor/google.golang.org/api/internal/BUILD.bazel` again :(

Reply:
I still have no quick idea, will try to find some time to have look tomorrow

Reply:
I'm currently reading up on [gazelle dependency resolution](https://github.com/bazelbuild/bazel-gazelle#dependency-resolution), trying out the `# gazelle resolve` directive might be a way.

Reply:
Looks like [this](https://github.com/kubevirt/project-infra/pull/146/commits/90f483077e7f0e09cb3d3baaa25c434cd73ceabf#diff-0ea22445d900db774c8deb00f4a4df55R4) fixed it. Will need to read up why gazelle added some crlf to the yaml.

Reply:
Noticed it's not `gazelle`, the file is retrieved like this when using `go mod vendor`, so leaving it like this for the moment.

Reply:
AFAIU this is a file that is needed for [appveyor](https://www.appveyor.com/) under Windows, AFAIR the `crlf` is default there, so I guess this is fine.